src/d3d12/d3d12_renderer.cpp

```diff
@@ -332,9 +332,9 @@ namespace wr
 
 	void D3D12RenderSystem::StopRenderTask(CommandList* cmd_list, std::pair<RenderTarget*, RenderTargetProperties> render_target)
 	{
-		auto n_cmd_list = static_cast<D3D12CommandList*>(cmd_list);
-		auto n_render_target = static_cast<D3D12RenderTarget*>(render_target.first);
-		auto frame_idx = GetFrameIdx();
+		auto n_cmd_list = static_cast<d3d12::CommandList*>(cmd_list);
+		auto n_render_target = static_cast<d3d12::RenderTarget*>(render_target.first);
+		unsigned int frame_idx = GetFrameIdx();
```

Comment:
Why did you change this?

Reply:
My bad, changed that when I had some unexplainable errors which where suppressed by specifically specifying a type. Later it turned out to be caused by a different error. I’ll change this back in my next submit!